OutOfSchool/OutOfSchool.WebApi/Services/ProviderService.cs

```diff
@@ -269,6 +327,93 @@ public async Task<ProviderDto> Update(ProviderDto providerDto, string userId)
             }
         }
 
+        /// <inheritdoc/>
+        public async Task<ProviderDto> UpdateV2(ProviderDto providerDto, string userId)
```

Comment:
Please check if there are possibilities to avoid code duplication in `Update` and `UpdateV2`